src/organisms/cards/PlaybackCardWrapper/index.js

```diff
@@ -11,16 +13,33 @@ function PlaybackCardWrapper(props) {
     className,
     logo,
     amount,
+    unit,
     footer,
     footerOnClick,
   } = props;
 
+  const renderHeader = () => {
+    if (!logo || !amount) return null;
+
+    return (
+      <Layout
```

Comment:
it looks like the padding is a little off on the playback card wrapper - should be ru(1.5), and then there should be a little more padding under the logo